packages/client/tests/functional/_utils/setupTestSuiteEnv.ts

```diff
@@ -76,7 +76,18 @@ async function copyPreprocessed(from: string, to: string, suiteConfig: Record<st
  * @returns
  */
 function evaluateMagicComment(conditionFromComment: string, suiteConfig: Record<string, string>): boolean {
-  const script = new Script(conditionFromComment)
+  const script = new Script(`
+  const Providers = {
+    SQLITE: 'sqlite',
+    POSTGRESQL: 'postgresql',
+    MYSQL: 'mysql',
+    MONGODB: 'mongodb',
+    COCKROACHDB: 'cockroachdb',
+    SQLSERVER: 'sqlserver',
+  }
```

Comment:
Could this be moved into the context instead so we can benefit from typesafety (eg. for when we add more providers)?

Reply:
This is a quick hack, I'm happy if you can suggest a code change for this. 😅